meson_options.txt

```
@@ -19,5 +19,6 @@ option('skip_introspection', type : 'boolean', value : false)
 option('test_dirty_git', type : 'boolean', value : false)
 option('test_installed_lib', type : 'boolean', value : false)
 option('with_docs', type : 'boolean', value : true)
+option('with_manpage', type : 'boolean', value : true)
```

```suggestion
option('with_manpage', type : 'feature', value : 'auto')
```
If we make this a `feature` option, you can just use it in the `required:` argument to `find_program()` and it will Do The Right Thing. (In the case of 'auto', it will return the program or a `disabler()` object if it cannot be found. It can also be passed `'enabled'` or `'disabled'` to force the result as appropriate.